### DISCUSSION TITLE
How to dock a nanotube with small molecule?

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#1048] [Q&A]
Hi.

I am trying to search for the best interaction configuration between a nanotube and a small molecule. 

As the nanotube is from a periodic system (and I will go back to it), I fixed all its atoms. In the case of the small molecule, in order to keep its integrity, I constrained all its atoms.

My issue is that all the generated complexes are 99% with the small molecule inside the nanotube, very rarely outside. This could be possible, of course, but even when I increase the number of optimized structures to be output, there is no configuration with the small molecule in the nanotube outside region (I need to run several times to get one complex with the molecule outside the nanotube).

As I am using default parameters, how can I be sure that the results are what they are and not due to bad initialization or bad parameters?

**Comment by marcelmbn** [comment]
@cplett I'm pretty sure you're the best to help with this issue.
